controller/router.js

```diff
@@ -83,7 +83,9 @@ module.exports = [
             validate: {
                 payload: {
                     stageId: Joi.number().integer(),
-                    trialId: Joi.number().integer()
+                    trialId: Joi.number().integer(),
+                    patStartDate: Joi.date().min(moment().format('YYYY-MM-DD')),
+                    patEndDate: Joi.date().min(moment().format('YYYY-MM-DD'))
```

Comment:
naming these `dateStarted` and `dateCompleted` would be clearer on how this maps to db.